devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -210,13 +209,15 @@ def configure() {
 	def cmds = refresh() +
 		zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, 30, 60 * 5, null) +
 		zigbee.batteryConfig() +
-		zigbee.temperatureConfig(30, 60 * 30) +
+        zigbee.temperatureConfig(30, 60 * 30) +
```

Comment:
This transition from tabs to spaces here is unnecessary

Reply:
Fixed